src/index.html

```diff
@@ -18,5 +22,85 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="/apple"
+              class="nav_link is-active"
+            >
+              APPLE
+            </a>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to simplify reading. There should be an empty line before and after each `li` block.